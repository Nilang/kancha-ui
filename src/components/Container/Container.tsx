import * as React from 'react'
import { View, Text } from 'react-native'

interface ContainerProps {
  param: string
}

const hello = 'HELLO'

const Container: React.FC<ContainerProps> = props => {
  return (
    <View>
      <Text>It's Alive</Text>
    </View>
  )
}

export default Container
