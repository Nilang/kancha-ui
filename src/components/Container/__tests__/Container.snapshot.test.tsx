import React from 'react'
import { render } from 'react-native-testing-library'

import Container from '../Container'

describe('Container', () => {
  it('renders correctly', () => {
    const tree = render(<Container />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
