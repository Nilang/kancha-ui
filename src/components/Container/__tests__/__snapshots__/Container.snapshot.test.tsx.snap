// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly 1`] = `
<View
  pointerEvents="auto"
  style={
    Object {
      "alignItems": undefined,
      "backgroundColor": undefined,
      "borderRadius": undefined,
      "flex": undefined,
      "flexDirection": undefined,
      "height": undefined,
      "justifyContent": undefined,
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "padding": undefined,
      "paddingBottom": undefined,
      "paddingHorizontal": undefined,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": undefined,
      "width": undefined,
    }
  }
/>
`;
